src/ILCompiler.Compiler/src/Compiler/RyuJitCompilationBuilder.cs

```diff
@@ -27,7 +27,7 @@ public RyuJitCompilationBuilder(CompilerTypeSystemContext context, CompilationMo
         {
         }
 
-        public override CompilationBuilder UseBackendOptions(IEnumerable<string> options)
+        public static KeyValuePair<string, string>[] ParseJitOptions(IEnumerable<string> options)
```

Comment:
Is this functionality interesting enough that we would want to share this? (I would eventually like to kick out CoreRT-RyuJIT compiler out of ILCompiler.Compiler (in the same way CppCodegen/Wasm is already out). Adding this dependency will just add work in moving it out.

Reply:
I can remove the dependency, sure. And yeah, I might end up moving ryujit out as part of this work if the line between CoreRT Jit and CoreCLR R2R Jit becomes an annoyance